packages/p2p/src/components/orders/order-table/order-table-row.jsx

```diff
@@ -68,51 +90,93 @@ const OrderRow = observer(({ row: order }) => {
 
     const offer_amount = `${amount_display} ${account_currency}`;
     const transaction_amount = `${price_display} ${local_currency}`;
+    const is_buy_order_type_for_user = (is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad);
+    const order_type = is_buy_order_type_for_user ? localize('Buy') : localize('Sell');
 
     return (
-        <div onClick={() => order_store.setQueryDetails(order)}>
-            <Table.Row
-                className={classNames('orders__table-row orders__table-grid', {
-                    'orders__table-grid--active': general_store.is_active_tab,
-                    'orders__table-row--attention': !isOrderSeen(id),
-                })}
-            >
-                <Table.Cell>
-                    {(is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad) ? (
-                        <Localize i18n_default_text='Buy' />
-                    ) : (
-                        <Localize i18n_default_text='Sell' />
-                    )}
-                </Table.Cell>
-                <Table.Cell>{id}</Table.Cell>
-                <Table.Cell>{other_user_details.name}</Table.Cell>
-                <Table.Cell>
-                    <div
-                        className={classNames('orders__table-status', {
-                            'orders__table-status--danger': should_highlight_danger,
-                            'orders__table-status--alert': should_highlight_alert,
-                            'orders__table-status--success': should_highlight_success,
-                            'orders__table-status--disabled': should_highlight_disabled,
+        <React.Fragment>
+            {isMobile() ? (
```

Comment:
Same here, you can get rid of ternary statements and just use an if statement instead.